topics/single-cell/tutorials/bulk-music-3-preparebulk/tutorial.md

```diff
@@ -1,9 +1,9 @@
 ---
 layout: tutorial_hands_on
-subtopic: deconvo
-priority: 3
+subtopic: datamanipulation
```

Comment:
I guess the subtopic fits, but we're using deconvolution specific tools thought right? I get the impression that datamanipulation would be more for general datasets like tabular

Reply:
Oh that's a good point. Maybe we need to call it something better, like Single cell data reformatting?